hydra-cardano-api/src/Hydra/Cardano/Api/Tx.hs

```diff
@@ -115,14 +111,18 @@ fromLedgerTx ledgerTx =
   scripts =
     Map.elems $ Ledger.txscripts' wits
 
+  scriptsData :: TxBodyScriptData Era
   scriptsData =
     TxBodyScriptData
-      ScriptDataInBabbageEra
+      AlonzoEraOnwardsBabbage
       (Ledger.txdats' wits)
       (Ledger.txrdmrs' wits)
 
   validity = case isValid of
     Ledger.IsValid True ->
-      TxScriptValidity TxScriptValiditySupportedInBabbageEra ScriptValid
+      TxScriptValidity AlonzoEraOnwardsBabbage ScriptValid
     Ledger.IsValid False ->
-      TxScriptValidity TxScriptValiditySupportedInBabbageEra ScriptInvalid
+      TxScriptValidity AlonzoEraOnwardsBabbage ScriptInvalid
+
+makeShelleyKeyWitness :: TxBody Era -> ShelleyWitnessSigningKey -> KeyWitness Era
+makeShelleyKeyWitness = Cardano.Api.makeShelleyKeyWitness ShelleyBasedEraBabbage
```

Comment:
Should: move this to `Hydra.Cardano.Api` which is the intended home for this `Era`-specific functions and patterns (see README.md and module haddocks)

This way, you also do not need to qualify the `makeShelleyKeyWitness` because these sub-modules won't have the top-level `Api.hs` in scope.

Reply:
Done in https://github.com/input-output-hk/hydra/commit/67e685a529ba4aea9cefaecfe358ba8604a9634e